README.md

```diff
@@ -2403,6 +2403,10 @@ in `conf/training/gpt3/5b.yaml`. The parameters can be modified to adjust the
 hyperparameters of the training runs. All supported model types and sizes can be found
 in `conf/training` folder.
 
+We support changeable global batch size during training. It can be set by changing `rampup_batch_size` parameter under the training config. Should be a list of 3 values: `[<start_batch_size>, <batch_size_increment>, <rampup_samples>]`<br>.
+Example: `rampup_batch_size=[256, 128, 50000000]`<br>.
+In case of using ramp up batch size, nodes shceduler will be created. It allows use smaller number of nodes for smaller batch size stages. Nodes scheduler will be automatically according to `training.trainer.num_nodes` parameter. Please, note that ramp up batch size only works with fused_adam optimizer for now.
```

Comment:
```suggestion
In case of using ramp up batch size, nodes scheduler will be created. It allows the use of a smaller number of nodes for smaller batch size stages. Nodes scheduler will be created automatically according to `training.trainer.num_nodes` parameter. Please, note that ramp up batch size only works with fused_adam optimizer for now.
```

Reply:
It's not clear to me what num_nodes should be set to? Should it be set to the maximum number that you want to use?

Reply:
If it only works with fused_adam does nemo toolkit raise an error if another optimizer is used?

Reply:
@ericharper 
1) yes, it related to the maximum number of nodes we want to use. I'll add this to readme.
2) do you mean we have to add assertion to NeMo if we don't use fused_adam optimizer with ramp up bs?